droplets.go

```diff
@@ -33,6 +34,7 @@ var _ DropletsService = &DropletsServiceOp{}
 // Droplet represents a DigitalOcean Droplet
 type Droplet struct {
 	ID          int       `json:"id,float64,omitempty"`
+	Names       []string  `json:"names,omitempty"`
```

Comment:
Yes, you send `Names` instead of `Name` with `MultipleCreate` because that's the API is set up.